sklearn/cross_decomposition/pls_.py

```diff
@@ -505,6 +516,24 @@ class PLSRegression(_PLS):
 
     Notes
     -----
+    We denote :
+        T: x_scores_
+        U: y_scores_
+        W: x_weights_
+        C : y_weights_
+        P : x_loadings_
+        Q : y_loadings__
+
+    Computed such that:
+        X = T P^T + Err and Y = U Q^T + Err
```

Comment:
so ``T`` is ``x_scores_`` and ``^T`` is transpose? Is the ``T`` standard PLS notation? the mixture of python syntax (for the slicing) and latex syntax (for the ``^T``) is also a bit confusing. Maybe ``.T``?